core/tauri/src/manager.rs

```diff
@@ -1102,12 +1102,13 @@ impl<R: Runtime> WindowManager<R> {
     S: Serialize + Clone,
     F: Fn(&Window<R>) -> bool,
   {
+    let emit_args = WindowEmitArgs::from(event, source_window_label, payload)?;
     assert_event_name_is_valid(event);
     self
       .windows_lock()
       .values()
       .filter(|&w| filter(w))
-      .try_for_each(|window| window.emit_internal(event, source_window_label, payload.clone()))
```

Comment:
I don't think this clone was ever needed. Even when we serialize once per window, it's just creating a string, so using `&payload` would have been better.